### PR TITLE
More caveat consistency tests

### LINKED ISSUE #1105: Add additional consistency tests for caveats
We should have an exhaustive set of consistency tests for caveats

### PR DESCRIPTION
**Pull request by josephschorr**
Fixes #1105

### CODE REVIEW

File: internal/services/integrationtesting/testconfigs/expiration.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+schema: |+
+  definition user {}
+
+  caveat is_not_expired(current_time timestamp, expiration timestamp) {
+    current_time < expiration
+  }
+
+  definition document {
+  	relation viewer: user with is_not_expired
```
Comment (vroldanbet) [inline internal/services/integrationtesting/testconfigs/expiration.yaml:10]: fix indent

Comment (vroldanbet) [inline internal/services/integrationtesting/testconfigs/expiration.yaml:10]: fix indent

File: internal/services/integrationtesting/testconfigs/quay.yaml [hunk 2]
```diff
@@ -1,86 +1,101 @@
...
+  definition user {}
 
-  definition quay/namespace {
-  	relation parent: quay/namespace | quay/organization
-  	relation admin: quay/user
-  	relation creator: quay/user
+  definition anonymoususer {}
+
+  caveat is_not_geo_banned(user_ip ipaddress, cidr string) {
+    !user_ip.in_cidr(cidr)
```
Comment (vroldanbet) [inline internal/services/integrationtesting/testconfigs/quay.yaml:8]: fix indent wrt definition

**Review by vroldanbet — APPROVED** [review]
just nits

**Comment by josephschorr** [comment]
Fixed indentation

**Review by vroldanbet — APPROVED** [review]

**Pull request merged**